### PR TITLE
Hello Micoverse Project

### PR DESCRIPTION
**Pull request by jaamanu**
Added the following index.html 
Added three different linters
created a .githubflow folder for linters
created .gitognore file and ignore the test.md and node_module file
created test.md file 
created style.css file and styled my h1 tag
updated my README.md file with ny information 
I added and committed each file created and pushed it to my repo branch.
made comments and created a pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
+- github
+
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```
Comment (zieeco) [inline README.md]: - Kindly remove this text block as these are instructions meant for you to follow to add instructions on how the users could have your project on their local machine.  Update to this: 
`To get a local copy up and running follow these simple example steps.`
Then you can now add your steps..

File: README.md [hunk 2]
```diff
@@ -1,2 +1,61 @@
...
+VS code
+linters
+browser
+
+### Setup
+copy the url to your local https://github.com/jaamanu/Project.git
+
+## JAMES ASIBEY MANU
+
+👤 **Author1**
```
Comment (zieeco) [inline README.md]: - Kindly change this to Author as you've worked on this project alone. Denoting it with Author1 simply means there are other authors that have collaborated on this project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,61 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+
```
Comment (zieeco) [inline README.md]: - Kindly add the missing content in the "Acknowledgment" header section to make your readme more professional.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,61 @@
...
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (zieeco) [inline README.md:48]: - Please, kindly add the missing licensed content as the link is broken to make your readme more professional

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @<STUDENT NAME>,


Great job so far on setting up the Hello Microverse Project. 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️

- [x] Kindly update the PR title to `Hello Microverse Project` as yours now is not readable to the users. You can add a space after the text "Hello" to make it clear and readable.

_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @jaamanu, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] Kindly make all the changes asked by the previous reviewer-:
1. Kindly update the Pull Request title to Hello Microverse Project.
2. Make changes to readme as asked previously: [1st change](https://github.com/jaamanu/Project/pull/1#discussion_r946957726), [2nd change](https://github.com/jaamanu/Project/pull/1#discussion_r946959785), [3rd change](https://github.com/jaamanu/Project/pull/1#discussion_r946961432), [4th change](https://github.com/jaamanu/Project/pull/1#discussion_r946963799)

- [ ] You have made changes to the HTML file and changed to Hello Microverse Project instead you have to change the pull request title.
- [ ] Kindly make sure linters are setup properly and all are in green check mark. You can copy all the file from [here](https://github.com/microverseinc/linters-config/tree/master/html-css) except the readme.
- [ ] You can view this to make changes [here](https://www.loom.com/share/e41329c7187645afa576eaafab5a9a53)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaamanu** [comment]
Thank for the update will update all mentioned above. i didnt understand that PR meant for pull request.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @jaamanu               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the changes requested by the previous reviewers good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**